include/seqan3/range/metafunction.hpp

```diff
@@ -54,78 +58,401 @@ namespace seqan3
  */
 
 // ----------------------------------------------------------------------------
+// uncvref
+// ----------------------------------------------------------------------------
 
-/*!\brief Whether a type has a `value_type` member and/or `ranges::value_type_t<t>` resolves [Value metafunction].
- * \tparam t The type to recurse on; must have `ranges::value_type_t<t>`
- * \hideinitializer
+/*!\brief Return the input type with references removed [Type metafunction].
+ * \tparam t The type to operate on.
+ */
+template <typename t>
+using unref_t = std::remove_reference_t<t>;
+
+// ----------------------------------------------------------------------------
+// uncvref
+// ----------------------------------------------------------------------------
+
+/*!\brief Return the input type with `const`, `volatile` and references removed [Type metafunction].
+ * \tparam t The type to operate on.
+ */
+template <typename t>
+using uncvref_t = std::remove_const_t<std::remove_volatile_t<std::remove_reference_t<t>>>;
```

Comment:
please name it `remove_cvref`

http://en.cppreference.com/w/cpp/types/remove_cvref (since c++20)